From: Simon McVittie <smcv@debian.org>
Date: Fri, 15 Apr 2022 11:38:23 +0100
Subject: Ignore result of test-network-panel

This is a workaround for the test intermittently failing, especially
on slower architectures. Continue to run it with the result ignored,
and force its output to be logged, so that we can see how often it
fails in practice.

Bug: https://gitlab.gnome.org/GNOME/gnome-control-center/-/issues/1768
Forwarded: not-needed, Debian-specific workaround
---
 tests/network/meson.build           | 1 +
 tests/network/test-network-panel.py | 3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/network/meson.build b/tests/network/meson.build
index 34f7c74..203af9d 100644
--- a/tests/network/meson.build
+++ b/tests/network/meson.build
@@ -29,6 +29,7 @@ test(
   'test-network-panel',
   find_program('test-network-panel.py'),
       env : envs,
+  verbose : true,
   timeout : 60
 )
 
diff --git a/tests/network/test-network-panel.py b/tests/network/test-network-panel.py
index 1098bab..dac0894 100644
--- a/tests/network/test-network-panel.py
+++ b/tests/network/test-network-panel.py
@@ -41,4 +41,5 @@ class PanelTestCase(X11SessionTestCase, GTest):
 
 if __name__ == '__main__':
     # avoid writing to stderr
-    unittest.main(testRunner=unittest.TextTestRunner(stream=sys.stdout, verbosity=2))
+    # TODO: result is ignored: https://gitlab.gnome.org/GNOME/gnome-control-center/-/issues/1768
+    unittest.main(testRunner=unittest.TextTestRunner(stream=sys.stdout, verbosity=2), exit=False)
